cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -281,6 +286,16 @@ SSHPublicKey getPublicKey() {
       .map(String::trim).collect(Collectors.toList());
   }
 
+  @Nullable
+  Boolean isAutoPolicyEnabled() {
```

Comment:
Why does it need to be `Nullable`. Can it not by default also return `false` if not set.

Reply:
Yes agree. Its set to false by default.